modules/persistent-storage-csi-vol-populator-procedure-admin.adoc

```diff
@@ -0,0 +1,192 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vol-populators.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-vol-populator-procedure-admin_{context}"]
+= Creating CRDs for volume populators
+
+The following procedure explains how to create an example "hello, world" custom resource definition (CRD) for a volume populator.
+
+Users can then create instances of this CRD to populate persistent volume claims (PVCs).
+
+.Prerequisites
```

Comment:
How about highlighting the fact that users have to develop their own populator? It's basically a prerequisite for everything in this doc if they want something else than "hello world". The image reference below `registry.k8s.io/sig-storage/hello-populator:v1.0.1` is where users would put their image with (custom) populator logic.

Also, can we link to upstream repos? If we can I believe it would help users if we guide them towards the code of this "hello world" populator so they can see how it's done and could create their own. The implementation is here: https://github.com/kubernetes-csi/lib-volume-populator/tree/master/example/hello-populator